drape_frontend/backend_renderer.cpp

```diff
@@ -70,7 +71,7 @@ BackendRenderer::BackendRenderer(Params && params)
                               make_unique_dp<FlushSubrouteMarkersMessage>(std::move(subrouteMarkersData)),
                               MessagePriority::Normal);
   });
-
+  
```

Comment:
nit: Setup your editor to remove trailing whitespaces.

Reply:
Fixed